### ISSUE TITLE
Promise: List Component - List view (stretch goal)

### ISSUE THREAD

**Issue by mixth** [wevisdemo/parliament-watch#103]
The PR shall be merged into `promise/main` branch.

This issue is blocked by:
- #96 

# Purpose
To show promises in a different presentation

# Note
- This is a stretch goal issue. We do not need this on our launch date. Please consider this as a low priority item.

# UI
- Add an icon to switch between card / list view (top right)
Please refer to [latest UI in Figma](https://www.figma.com/design/ywiKyd0XIgDXf8crX4aoYE/Design_Phase1-2-%5BPromise-Tracker%5D?node-id=4949-11913&node-type=frame&t=HmpwBOeDR4PxrdVq-0) from the design team.

![list view](https://github.com/user-attachments/assets/a5eed6cc-5d39-48ad-868b-3dfff80320a1)

**Comment by Th1nkK1D** [comment]
Merged #110
